tembo-operator/src/cloudnativepg/hibernate.rs

```diff
@@ -193,3 +208,98 @@ pub async fn reconcile_cluster_hibernation(cdb: &CoreDB, ctx: &Arc<Context>) ->
     }
     Ok(())
 }
+
+async fn update_pooler_instances(
+    pooler: &Option<Pooler>,
+    cdb: &CoreDB,
+    ctx: &Arc<Context>,
+) -> Result<(), Action> {
+    let name = cdb.name_any();
+
+    match pooler {
+        Some(p) => {
+            let current_instances = p.spec.instances.unwrap_or(1);
+            let desired_instances = if cdb.spec.stop { 0 } else { 1 };
```

Comment:
Shouldn't this be a call to `get_pooler_instances`?

Reply:
Thanks I missed updating that.